<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ImageDownloader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/imagedownloader.qrc">
    <normaloff>:/images/github plaatje.png</normaloff>:/images/github plaatje.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="inputs" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>720</y>
      <width>1280</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>891</width>
       <height>35</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="queueLabel">
        <property name="text">
         <string>0 items in queue</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="blackListButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>BlackList</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="resBox">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <item>
         <property name="text">
          <string>Any</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1920x1080</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2560x1440</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3840x2160</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1080x1920</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1440x2560</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1080..x1920..</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1440..x2560..</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1920..x1080..</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2560..x1440..</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3840..x2160..</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="skipButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Next 5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="endButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>End</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>1091</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>                    Image Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="optionsButton">
     <property name="geometry">
      <rect>
       <x>1160</x>
       <y>42</y>
       <width>116</width>
       <height>27</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Options</string>
     </property>
    </widget>
    <widget class="QLabel" name="depthLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>41</y>
       <width>100</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>Depth: 0</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1280</width>
      <height>720</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1280</width>
      <height>720</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/imagedownloader.qrc"/>
 </resources>
 <connections/>
</ui>
